tests/test_eval.py

```diff
@@ -73,12 +73,14 @@ def test_javascript_specific_primitives(client):
 
 
 def test_javascript_noquery(client):
-    with raises(ValueError, match="No script found; must specify a javascript"):
+    with raises(
```

Comment:
Same point as above - I think these tests can be removed along with the None-checking for the `xquery` and `javascript` args, as that isn't done anywhere. And the message from the server is helpful for both errors.